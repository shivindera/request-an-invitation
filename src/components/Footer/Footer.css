.broccoli-footer {
  background: white;
  width: 100%;
  padding: 16px;
  border-top: 1px solid gray;
  position: fixed;
  bottom: 0;
}

.broccoli-footer-container {
  display: flex;
  flex-direction: column;
  align-items: center;
  width: 100%;
  max-width: 1024px;
  margin: 0 auto;
}

.broccoli-footer-heart {
  font-size: 20px;
  color: #669900;
}
