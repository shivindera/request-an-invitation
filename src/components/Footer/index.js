// Static Footer component

import './Footer.css';

const Footer = () => {
  return (
    <footer className='broccoli-footer'>
      <div className='broccoli-footer-container'>
        <p>
          Made with <span className='broccoli-footer-heart'>&#10084;</span> at Home
        </p>
        <p>&copy; 2022 Broccoli & Co. All rights reserved.</p>
      </div>
    </footer>
  );
};

export default Footer;
