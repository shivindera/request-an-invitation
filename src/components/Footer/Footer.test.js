import { render } from '@testing-library/react';
import Footer from './';

it('renders Footer as expected', () => {
  const footer = render(<Footer />);
  expect(footer.container).toMatchSnapshot();
});
