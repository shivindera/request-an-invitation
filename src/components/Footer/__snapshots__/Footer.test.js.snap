// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer as expected 1`] = `
<div>
  <footer
    class="broccoli-footer"
  >
    <div
      class="broccoli-footer-container"
    >
      <p>
        Made with 
        <span
          class="broccoli-footer-heart"
        >
          ❤
        </span>
         at Home
      </p>
      <p>
        © 2022 Broccoli & Co. All rights reserved.
      </p>
    </div>
  </footer>
</div>
`;
